src/NServiceBus.Core/Pipeline/Incoming/LoadHandlersConnector.cs

```diff
@@ -32,6 +36,11 @@ public override async Task Invoke(IIncomingLogicalMessageContext context, Func<I
                     throw new InvalidOperationException(error);
                 }
 
+                if (isDebugIsEnabled)
+                {
+                    LogHandlersInvocation(context, handlersToInvoke);
+                }
+
```

Comment:
Remove whitespace

Reply:
Sorry @andreasohlund, not sure what whitespace you're referring to.